### ISSUE TITLE
Cpu 

### ISSUE THREAD

**Issue by keko950** [AlexeyAB/darknet#3820]

**Comment by LukeAI** [comment]
I don't think so, but what's the problem with compiling darknet twice? libdarknet.so is only a few MB so compile two versions and load them both in your wrapper program

**Comment by Leo16112** [comment]
I would load classification using opencv, and detection with darknet with GPU.

**Comment by keko950** [comment]
> I don't think so, but what's the problem with compiling darknet twice? libdarknet.so is only a few MB so compile two versions and load them both in your wrapper program

Finally managed to setup two models in the gpu anyway :)
